pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# References <!-- Link to the Jira task -->
+[PROJ-XXX](https://frontkom.atlassian.net/browse/PROJ-XXX)
+
+### Changed <!-- What changed with this PR -->
+
+### Visuals <!-- Add here some screenshots -->
+
+### How can this be validated? <!-- Instructions for QA -->
+
+### Pull request checklist
+- [ ] Is it possible to deploy automatically? If it requires manual deployment, did you include the steps needed to deploy?
+- [ ] Did you make sure documentation have been updated, if needed?
+- [ ] Did you include all necessary translations, if any?
+- [ ] Did you make sure to update sanitization routines if fields for storing personal information are added?
```

Comment:
I like shorter texts in there, faster to read and therefore more likely ppl will actually spend 2 sec reading them. 😄 

```suggestion
- [ ] Can be deploy automatically? _If manual actions are required:_ did you describe the deploy steps?
- [ ] Is the documentation updated, if it makes sense? <!-- Check this also if no updating is needed -->
- [ ] Are necessary translations added/updated? <!-- Check this even if no translations are being changed -->
- [ ] Did you update sanitization routines, where personal information is handled?
```